warehouse/models/staging/transit_database/base/_base_transit_database.yml

```diff
@@ -1,6 +1,38 @@
 version: 2
 
 models:
+  - name: base_tdqi_gtfs_datasets_ct_gtfs_datasets_map
+    description: |
+      Mapping between two instances of synced `gtfs_datasets` table in Airtable;
+      one copy in California Transit base and the other in Transit
+      Transit Data Quality Issues base. Different Airtable internal record IDs
+      are assigned in each instance so this table maps the IDs between
+      instances, using `name` (Airtable primary field) to join.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tts_key
```

Comment:
I think that these should be `tdqi_key` and `tdqi_date` -- the prefix is the base abbreviation

Reply:
Resolved in [ce2d2e4](https://github.com/cal-itp/data-infra/pull/2781/commits/ce2d2e4af50b81448381ff68ec3c28a31913732c)